md/tomcat-bundle.md

```diff
@@ -116,8 +116,9 @@ When you create a tenant, the tenant administrator is created with the default u
 Change these tenant-specific credentials for an existing tenant by updating the `userName` and `userPassword` properties in `bonita-tenant-community-custom.properties`.
 
 ::: warning
-For the default tenant, the tenant administrator username and password must also be defined in file [`<TOMCAT_HOME>/setup/platform_conf/initial/platform_portal/platform-tenant-config.properties`](BonitaBPM_platform_setup.md), with exactly the same values that you set in `bonita-tenant-community-custom.properties`.
-At platform creation, this file contains the default username and password for the default tenant. 
+For the **default tenant**, the tenant administrator username and password must also be changed in file [`<TOMCAT_HOME>/setup/platform_conf/initial/platform_portal/platform-tenant-config.properties`](BonitaBPM_platform_setup.md),
+with exactly the same values that you set in `bonita-tenant-community-custom.properties` (see above). At platform creation, this file contains the default username and password for the default tenant.
+More detailed understanding of [Tenant administrator credentials](tenant_admin_credentials.md).
```

Comment:
Perhaps you could rephrase the last sentence:

"For further details and a better understanding, please read the section [Tenant administrator credentials](tenant_admin_credentials.md)."

Reply:
I'll do that.